app/views/wyatt_controller/index.html.erb

```diff
@@ -1 +1,12 @@
-<h1>Hi, James!</h1>
+<!DOCTYPE html>
```

Comment:
FYI, the `yield` call in the application layout file is where the view file code gets injected.